docs/fuse-box/tutorials/api-trade-quotes.mdx

```diff
@@ -0,0 +1,334 @@
+---
+title: REST API - Trade v2 GET Quotes 
+sidebar_position: 17
+---
+
+Trade v2 API
+
+# Introduction
+
+In this tutorial, we'll walk through using the FuseBox REST API in a Next.js application to consume the Trade v2 API **”[Get a quote for buying or selling any ERC20 token](../../trade-api/get-quote)”** endpoint. We'll utilize React hooks to manage the State and display the response in a table format.
+
+## Prerequisites
+Before starting, ensure you have the following:
+
+Node.js installed on your machine.
+- Code Editor: Use your preferred code editor; VS Code is recommended.
+- An EOA wallet with a private key. You can use an existing one or create a new wallet.
+- A basic understanding of React.js and Next.js.
+- An API key from the Fuse Console. Get one [here](https://console.fuse.io/build).
+
+## Step 1: Set Up a Next.js Project
+If you haven't already set up a Next.js project, you can create one using the following commands:
+
+```bash
+npx create-next-app my-project
+cd my-project
+```
+
+Answer the required prompts from NextJS in the terminal. Note that we use TypeScript and Tailwind CSS for this application.
+
+## Step 2: API Endpoint
+In this example tutorial, we are going to build a Form element that takes in Tokens for BUY and SELL orders and an amount and returns the response in a table. We will use the Axios library to call the Fuse REST APIs and return the responses.
+
+The endpoint that we will use in this tutorial is the Get List of Tokens Owned by Address.
+
+The base URL is https://api.fuse.io/api/v1/trade/quote, and it takes three required parameters: **buyToken**, **sellToken**, and **amount**. You can get an API key from [Console](https://console.fuse.io/build).
+
+The URL, when appended with the parameters, for example:
+
+https://api.fuse.io/api/v1/trade/quote?apiKey=YOUR_API_KEY&sellToken=FUSE&buyToken=0x34Ef2Cc892a88415e9f02b91BfA9c91fC0bE6bD4&sellAmount=1000000000000000000
+## Step 2: Install Axios
+Axios is a promise-based HTTP client for the browser and Node.js. Install Axios to handle the API requests:
+```bash
+npm install axios
+```
+
+## Step 3: Fetch the API response.
+
+In this step, we will test calling the API and parsing the response to the web browser console. Add the following code to the `index.tsx` file before the return statement.
```

Comment:
As a result of lack of instructions, in my case index.tsx wasnt created at all.

Reply:
Fixed.